packages/svgs/src/scripts/load-icons.ts

```diff
@@ -0,0 +1,93 @@
+import * as path from "path";
+import * as fs from "fs";
+import camelCase from "camelcase";
+import {
+    ICONS_SIZES 
+} from "./constants.ts";
+import { getAllFiles } from "./helper.ts";
+
+interface DataItem {
+    name: string;
+    group: number;
+}
+
+export interface Icon {
+    content: string;
+    group: number;
+    filePath: string;
+    name: string;
+}
+
+const getFiles = (dir: string): Promise<string[]> => {
+    return new Promise((resolve, reject) => {
+        if (fs.existsSync(dir)) {
+            resolve(getAllFiles(dir));
+        } else {
+            reject(`no such directory ${dir}`);
+        }
+    });
+};
+
+const checkSameName = (data: DataItem[]): void => {
+    const sizes = ICONS_SIZES;
+    const dataGroupedBySize = sizes.map(s => data.filter(d => d.group === s));
+  
+    dataGroupedBySize.forEach(groupedData => {
+        const unique = [...new Set(groupedData.map(d => d.name))];
+  
+        const lookup = groupedData.reduce((a: Record<string, number>, e) => {
```

Comment:
```
        const lookup = groupedData.reduce((a , e) => {
            a[e.name] = ++a[e.name] || 0;

            return a;
        }, {} as Record<string, number>);
```

type the {} instead of forcing a's type from  unknown to Record